app/scripts/modules/amazon/src/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -30,10 +30,10 @@ describe('awsServerGroupCommandBuilder', function () {
 
   describe('buildNewServerGroupCommand', function () {
     it('initializes to default values, setting usePreferredZone flag to true', function () {
-      var command = null;
+      let command = null;
       AWSProviderSettings.defaults.iamRole = '{{application}}IAMRole';
       this.awsServerGroupCommandBuilder
-        .buildNewServerGroupCommand({ name: 'appo', defaultCredentials: {}, defaultRegions: {} }, 'aws')
+        .buildNewServerGroupCommand({ name: 'appo', defaultCredentials: {}, defaultRegions: {}, attributes: {} }, 'aws')
```

Comment:
Fixing failure seen locally:
```

FAILED TESTS:
  awsServerGroupCommandBuilder
    buildNewServerGroupCommand
      ✖ initializes to default values, setting usePreferredZone flag to true
      TypeError: Cannot read property 'createTs' of undefined
```

See [LOC](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/amazon/src/serverGroup/configure/serverGroupCommandBuilder.service.js#L120) where `createTs` is used.

Reply:
You can also use the `applicationModelBuilder.createApplicationForTests` to make sure all the attributes you need are stubbed out.

Reply:
`attributes: {}` is now removed.